Source/WebCore/page/NavigationInterceptHandler.idl

```diff
@@ -26,5 +26,6 @@
 [
     EnabledBySetting=NavigationAPIEnabled,
     GenerateIsReachable=ImplScriptExecutionContext,
-    SkipCallbackInvokeCheck
+    SkipCallbackInvokeCheck,
+    RethrowException
```

Comment:
Could you clarify why we need RethrowException here? Seems odd that we need any custom IDL attribute to get standard behavior.

Reply:
I misread the spec, I thought exceptionBehavior could be two values with a default of "rethrow",. but failed to read the sentence about the special handling when callable’s return type is a promise type, basically ignoring exceptionBehavior.